test/AutoDiff/floating_point.swift.gyb

```diff
@@ -0,0 +1,36 @@
+// RUN: %empty-directory(%t)
+// RUN: %gyb %s -o %t/floating_point.swift
+// RUN: %target-build-swift %t/floating_point.swift -o %t/a.out
+// RUN: %target-run %t/a.out
```

Comment:
I wonder if `%target-run-simple-swiftgyb` works? It's a one-liner.
```suggestion
RUN: %target-run-simple-swiftgyb 
```

Reply:
Yes, this works. Done!